src/screens/onboarding/Onboarding.tsx

```diff
@@ -64,27 +60,34 @@ export const OnboardingScreen = () => {
   const BackButton = <Button text={i18n.translate('Onboarding.ActionBack')} variant="subduedText" onPress={prevItem} />;
   const LanguageButton = <LanguageToggle />;
 
+  const [layout, setLayout] = useState<LayoutRectangle | undefined>();
+  const onLayout = useCallback(({nativeEvent: {layout}}: LayoutChangeEvent) => {
+    setLayout(layout);
+  }, []);
+
   return (
     <Box flex={1} backgroundColor="overlayBackground">
-      <Box position="absolute" bottom={50 + insets.bottom} width="100%" opacity={0.4}>
-        <OnboardingBg width="100%" viewBox="0 0 375 325" />
-      </Box>
       <SafeAreaView style={styles.flex}>
         <Header isOverlay />
-        <Carousel
-          ref={carouselRef}
-          data={contentData}
-          renderItem={renderItem}
-          sliderWidth={viewportWidth}
-          itemWidth={viewportWidth}
-          onSnapToItem={newIndex => setCurrentIndex(newIndex)}
-        />
-        <Box alignItems="center" justifyContent="center" flexDirection="row">
-          <Box flex={1} paddingHorizontal="l" paddingBottom="l">
-            {isStart ? LanguageButton : BackButton}
+        <Box flex={1} justifyContent="center" onLayout={onLayout}>
```

Comment:
The only issue that I want to call out in this PR is that I have to use `onLayout` in order to make `Carousel` vertical center alignment. It will delay rendering up to a second depending on device as you may notice in debug build. On production build on same device, it is lightning fast. 

An alternative of this is to get viewPort height, subtract header and footer because we know their size. 

cc @jordanAtShopify What do you think?

Reply:
I feel like getting viewportHeight and subtracting from it would be better to avoid the delay, but if this works then it is fine I think.

Reply:
One thing to double check is if this onLayout works with the ScrollView and with landscape mode

Reply:
Landscape mode is different issue. So, I think we can address it in different PR. I will switch to viewPort subtraction 👍

Reply:
Ya for sure, I just meant if this makes landscape mode worse it is probably not a good idea as it will have to be changed soon

Reply:
On that note should probably use the dynamic `useWindowDimensions` hook to support layout changes

Reply:
Thanks for the tip 👍

Reply:
I now think we should keep onLayout for now until we saw any downside. The subtraction is kind of hacky.